### ISSUE TITLE
Fix jsx-a11y/click-events-have-key-events warnings raised by jsx-a11y linter.

### ISSUE THREAD

**Issue by yzen** [firefox-devtools/debugger#7538]
To get a list of accessibility warnings raised by jsx-a11y linter run: `yarn lint:jsx-a11y`.

From that list please note all instances that are flagged to have "jsx-a11y/click-events-have-key-events" problem. Each one needs to be addressed individually.

Once all of the instances are resolved make sure to change this setting in `.eslintrc.jsx-a11y` from:
```
"jsx-a11y/click-events-have-key-events": "warn",
```
to
```
"jsx-a11y/click-events-have-key-events": "error",
```

**Comment by Garbee** [comment]
[This error](https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/click-events-have-key-events.md) details the requirement that if you have an `onClick` assignment then you should also have a handler for keyboard events on the same node. This goes hand-in-hand with #7537.

I strongly recommend focusing on #7537 first, since doing that job well will naturally resolve this issue. Button nodes by default wire keyboard events to click events.

**Comment by janodvarko** [comment]
Firefox Developer Tools (including Debugger tool) development is entirely happening in Bugzilla and new issues should be reported here: https://bugzilla.mozilla.org/enter_bug.cgi?product=DevTools&component=Debugger

Also, supporting JSX in the code base is not on our long term roadmap, so this might be invalid report.

Honza
